public/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
+ "icons": [
+  {
+   "src": "\/android-icon-36x36.png",
```

Comment:
Yes manifest.json is for PWA's. We do not NEED the escaped slashes, however they can help if this gets embedded in a `<script>` tag. See https://stackoverflow.com/a/1580682/16626860. 

Since this is not going to be embedded, I will remove them for now.

Reply:
Updated ✅